src/bindings/python/wheel/setup.py

```diff
@@ -294,6 +294,16 @@ def configure(self, install_cfg):
                 ):
                     set_rpath(comp_data["rpath"], os.path.realpath(path))
 
+    def get_reallink(self, link_file):
+        real_name = os.readlink(link_file)
+        if not os.path.isabs(real_name):
+            real_name = os.path.join(os.path.dirname(link_file), real_name)
+        while Path(real_name).is_symlink():
+            real_name = os.readlink(real_name)
+            if not os.path.isabs(real_name):
+                real_name = os.path.join(os.path.dirname(link_file), real_name)
```

Comment:
maybe we need to emulate do / while loop here? because code block before and inside the loop are duplicated

Reply:
I have updated.